app/models/manageiq/providers/embedded_terraform/automation_manager/ibm_cloud_credential.rb

```diff
@@ -3,28 +3,26 @@ class ManageIQ::Providers::EmbeddedTerraform::AutomationManager::IbmCloudCredent
     {
       :component  => 'password-field',
       :label      => N_('IBM Cloud API Key'),
-      :helperText => N_('The API key for IBM Cloud. A value for this field is required if classic user name and classic API key are not provided. A valid connection must have value for IBM Cloud API Key or, IBM Cloud Classic Infrastructure User Name and IBM Cloud Classic Infrastructure API Key.'),
+      :helperText => N_('The API key for IBM Cloud. This credential requires either an IBM Cloud API Key or both a Classic Infrastructure User Name and Classic Infrastructure API key.'),
       :name       => 'auth_key',
       :id         => 'auth_key',
-      :type       => 'password',
-      :isRequired => true,
-      :validate   => [{:type => 'required'}],
+      :type       => 'password'
     },
   ].freeze
 
   EXTRA_ATTRIBUTES = [
     {
       :component  => 'text-field',
       :label      => N_('IBM Cloud Classic Infrastructure User Name'),
-      :helperText => N_('The User Name for IBM Cloud Classic Infrastructure. A value for this field is required when using classic IBM Cloud resources. A valid connection must have value for IBM Cloud API Key or, IBM Cloud Classic Infrastructure User Name and IBM Cloud Classic Infrastructure API Key.'),
+      :helperText => N_('The User Name for IBM Cloud Classic Infrastructure. This credential requires either an IBM Cloud API Key or both a Classic Infrastructure User Name and Classic Infrastructure API key.'),
       :name       => 'classic_user',
       :id         => 'classic_user',
       :maxLength  => 100,
     },
     {
       :component  => 'password-field',
       :label      => N_('IBM Cloud Classic Infrastructure API Key'),
-      :helperText => N_('The API key for IBM Cloud Classic Infrastructure A value for this field is required when using classic IBM Cloud resources. A valid connection must have value for IBM Cloud API Key or, IBM Cloud Classic Infrastructure User Name and IBM Cloud Classic Infrastructure API Key.'),
+      :helperText => N_('The API key for IBM Cloud Classic Infrastructure. This credential requires either an IBM Cloud API Key or both a Classic Infrastructure User Name and Classic Infrastructure API key.'),
```

Comment:
I don't like how these "what's required" are repeated.  @GilbertCherrie do we have a way to say we require either:

auth_key OR
(classic_user AND classic_key)

I think the `:validate => [{:type => 'required'}]` I removed above is to simplistic for this logic.

Reply:
Moving this to a followup PR with 2 credential types.